### PR TITLE
[java] Fix false positive in useTryWithResources when using a custom close method with multiple arguments

### LINKED ISSUE #1701: [java] UseTryWithResources does not handle multiple argument close methods
**Affects PMD Version:** 6.12

**Rule:** UseTryWithResources 

**Description:** We have a custom close method that we use closeLocalResources that can take in multiple parameters to close them. PMD is not flagging this as an issue when there are multiple parameters however it does successfully flag the issue when there is only 1 parameter in this custom close method.

**Code Sample demonstrating the issue:**

```
closeLocalResources(ps, rs); is not flagged while

closeLocalResources(ps); is flagged
```

**Running PMD through:** *[Ant]*

### PR DESCRIPTION
**Pull request by jainrish**
Fixes #1701 

Before submitting a PR, please check that:
 - [ ] The PR is submitted against `master`. The PMD team will merge back to support branches as needed.
 - [ ] `./mvnw clean verify` passes. This will [build](https://github.com/pmd/pmd/blob/master/BUILDING.md) and test PMD, execute PMD and checkstyle rules. [Check this for more info](https://github.com/pmd/pmd/blob/master/CONTRIBUTING.md#code-style)

**PR Description:**

### CODE REVIEW

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">This changeset introduces 0 new violations, 0 new errors and 0 new configuration errors,<br />
removes 0 violations, 2 errors and 2 configuration errors.<br />
<a href="http://chunk.io/pmd/072d4221d48b4908804a22342efa13aa/diff/index.html">Full report</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: pmd-java/src/main/resources/category/java/bestpractices.xml [hunk 1]
```diff
@@ -1546,7 +1546,7 @@ preserved.
 ][
     pmd-java:typeIs('java.lang.AutoCloseable')
     or
-    ../../PrimarySuffix/Arguments[@ArgumentCount = 1]//PrimaryPrefix[pmd-java:typeIs('java.lang.AutoCloseable')]
+    ../../PrimarySuffix/Arguments//PrimaryPrefix[pmd-java:typeIs('java.lang.AutoCloseable')]
```
Comment (jsotuyod) [inline pmd-java/src/main/resources/category/java/bestpractices.xml]: this actually checks at least 1 argument is an autoclosable… should we check **all** arguments are autoclosable by doing

```
not(../../PrimarySuffix/Arguments//PrimaryPrefix[not(pmd-java:typeIs('java.lang.AutoCloseable'))])
```

Comment (jainrish) [reply pmd-java/src/main/resources/category/java/bestpractices.xml]: @jsotuyod Actually, when I added the test case containing close method with multiple arguments, it was failing when [@ArgumentCount = 1], so I removed it. So, if we want to have a check for at least one argument in the close method, can we do [@ArgumentCount >= 1]? I checked and it is working.

Comment (jsotuyod) [reply pmd-java/src/main/resources/category/java/bestpractices.xml]: Nevermind… if the user configured the close method this is probably good enough

**Comment by adangel** [comment]
Thanks for your contribution! This will be part of PMD 6.13.0 :tada:

**Comment by jainrish** [comment]
@adangel @jsotuyod thanks for your support. I will be trying a few more beginner's issues before moving on to bigger ones to understand the code better. I hope that's okay.

**Pull request merged**